go/src/meguca/templates/body.go

```diff
@@ -527,7 +529,7 @@ func (c *bodyContext) parseCommands(bit string) {
 	case "8ball":
 		inner = append(inner, val.Eightball...)
 		c.state.iDice++
-	case "rcount":
+	case "pyu", "pcount", "rcount":
 		inner = strconv.AppendUint(inner, val.Pyu, 10)
```

Comment:
If the pyu setting was toggled between this post being closed and rendered, we will get a counter shift and runtime crashes. Need to invalidate this command, if val.Type is not correct. Do the same for the JS and C++ implementation, please.

Reply:
There doesn't seem to be a lot I can do about the JS and C++ implementation, I've tried about everything there is to try. The type in `c.Commands[c.state.iDice]` doesn't seem to equal whatever is in commandType. As well as a few other things I've tried, including raw numbers.
If you have an idea, or know what to do I'd appreciate it.

Reply:
Actually yeah, #pyu and #pcount don't change their types to the correct types in the server either, while #rcount does.
0 for pyu and pcount, 7 for rcount.

Reply:
What? Then what does it equal to?

Reply:
It's fine now, I wasn't assigning the type, I didn't consider my changes.
I appreciate the heads up. I'll see if the client and C++ client work now.